### PR TITLE
[JavaScript] strings - replace about.md file with concept files

### PR DESCRIPTION
**Pull request by ErikSchierboom**
In [this issue](https://github.com/exercism/v3/issues/2293) we're describing an evolution of the specification of this repo.
These changes include, amongst others, replacing the contents of the `after.md` document with individual concept documents. There are two documents per concept:

1. An `about.md` file containing the description of the concept
1. A `links.json` file containing a set of useful links related to the concept. 

This looks as follows in the repo:

```
<track>
├── concepts
│   ├── <concept>
│   │   ├── about.md
│   │   └── links.json
│   └── ...
```

This PR applies this change to this exercise by: 
                    
1. Copying the existing `after.md` file's contents to the `about.md` file of each concept listed in the exercise's `concepts` section in the `config.json` file.
1. Extract the links from the `after.md` file and put them into the `links.json` file

Before merging this PR, please check that the contents of the concept documents makes sense.
This is especially true when the exercise unlocks multiple concepts, as then the concept documents should be updated to only contain the information relevant to that concept.

Note: to make reviewing easier, the PR has been split into three separate commits. Please squash this PR upon merging.

### CODE REVIEW

File: languages/javascript/concepts/strings/about.md [hunk 1]
```diff
@@ -1 +1,5 @@
-TODO: add information on strings concept
+The key thing to remember about JavaScript strings is that we can call methods on them like `toUpperCase`, `trim`, etc.
```
Comment (SleeplessByte) [inline languages/javascript/concepts/strings/about.md:1]: Missing a lot of information on strings (currently in introduction)

**Review by SleeplessByte — CHANGES_REQUESTED** [review]

**Comment by ErikSchierboom** [comment]
As we move forward towards v3 launch, this PR now need merging as this PR makes your track adhere to the latest specification. Could you please check whether they’re ok and merge them? If you don’t have time, I’ll merge them mid-next week and we can tidy things up later.

File: languages/javascript/concepts/strings/about.md (hunk 1 above)
Comment (ErikSchierboom) [reply languages/javascript/concepts/strings/about.md]: I've added the information from the introduction.

**Review by SleeplessByte — APPROVED** [review]

**Pull request merged**